packages/snap-preact-components/src/components/Organisms/Autocomplete/Autocomplete.tsx

```diff
@@ -454,7 +470,33 @@ export const Autocomplete = observer((properties: AutocompleteProps): JSX.Elemen
 												</div>
 											))}
 										</div>
-									</>
+									</div>
+								) : null}
+
+								{history.length && !hideHistory ? (
+									<div className="ss__autocomplete__terms__history">
+										<div className="ss__autocomplete__title ss__autocomplete__title--history">
+											<h5>{historyTitle}</h5>
+										</div>
+										<div className="ss__autocomplete__terms__options ss__autocomplete__history__options">
+											{history.map((term: any) => (
```

Comment:
Can we remove the `any` type here to match `trending` and `terms` if it's not possible to use the `Term` type in all of these.